README.md

```diff
@@ -4,31 +4,36 @@ Read data from Victron devices over Bluetooth Low Energy.
 
 Some Victron devices broadcast some aspects of their state over Bluetooth on a regular basis. This crate makes it easy to access that data.
 
+This crate is intended to work on Linux and macOS devices. 
```

Comment:
Please remove this as it's not entirely accurate (embedded devices and Windows are supported with reduced features by compiling with no_std).

After merge I'll add something to the readme to make that clear.

Reply:
@felixwatts I am using this crate with great success together with embassy, esp-hal and trouble in a bare-metal  ESP32 no_std environment. Without the bluetooth feature at the moment, but an embassy/trouble implementation of `open_stream` would probably not require too much black magic.